tests/data/http_to_file_with_transform.json

```diff
@@ -2,7 +2,7 @@
   "data_dir": "/tmp/vector/",
   "sources": {
     "source_http": {
-      "type": "http",
+      "type": "http_server",
```

Comment:
The latest version of vector renames source `http` to `http_server`

Reply:
That will requires default config changes in stonewave too

Reply:
Just FYI